htdocs/admin/modules.php

```diff
@@ -666,6 +667,12 @@
         // Version
         print '<td class="center nowrap" width="120px">';
         print $versiontrans;
+        if (!empty($objMod->url_last_version)) {
+            $newversion = file_get_contents($objMod->url_last_version);
```

Comment:
To get external content, you must use method getURLContent of geturl.lib.php
Also making external accesses may creates pb to show the module page (when proxy or firewall disagree), making the module page very very slow or not possible to show, so introduce a hidden option to enable such feature.